c_common/models/Makefile

```diff
@@ -1,6 +1,8 @@
-BUILD_DIRS = command_sender_multicast_source reverse_iptag_multicast_source \
-	live_packet_gather chip_power_monitor data_speed_up_packet_gatherer \
-	extra_monitor_support
+BUILD_DIRS = compressor_bit_field_models \
```

Comment:
Why are the two `*_multicast_source` models not on the build list any more?

Reply:
have chekced and all algorithms have them. so they must have disappeared and then got readded. given this pr is dead. assuming nothing to worry about